sdk/table/azure-table/azure/table/_table_service_client.py

```diff
@@ -0,0 +1,268 @@
+            conn_str, credential, 'table')
+        if 'secondary_hostname' not in kwargs:
+            kwargs['secondary_hostname'] = secondary
+        return cls(account_url, credential=credential, **kwargs)
+
+    @distributed_trace
+    def get_service_stats(self, **kwargs):
+        # type: (...) -> dict
+        """Retrieves statistics related to replication for the Table service. It is only available on the secondary
+        location endpoint when read-access geo-redundant replication is enabled for the account.
+
+                :keyword callable cls: A custom type or function that will be passed the direct response
+                :return: Dictionary of Service Stats
+                :rtype: dict
+                :raises: ~azure.core.exceptions.HttpResponseError
+                """
+        try:
+            timeout = kwargs.pop('timeout', None)
+            stats = self._client.service.get_statistics(  # type: ignore
+                timeout=timeout, use_location=LocationMode.SECONDARY, **kwargs)
+            return service_stats_deserialize(stats)
+        except HttpResponseError as error:
+            process_storage_error(error)
+
+    @distributed_trace
+    def get_service_properties(self, **kwargs):
+        # type: (...) -> dict
+        """Gets the properties of an account's Table service,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+        :keyword callable cls: A custom type or function that will be passed the direct response
+        :return: Dictionary of service properties
+        :rtype: dict
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+        timeout = kwargs.pop('timeout', None)
+        try:
+            service_props = self._client.service.get_properties(timeout=timeout, **kwargs)  # type: ignore
+            return service_properties_deserialize(service_props)
+        except HttpResponseError as error:
+            process_storage_error(error)
+
+    @distributed_trace
+    def set_service_properties(
+            self,
+            analytics_logging=None,  # type: Optional[Any]
+            hour_metrics=None,  # type: Optional[Any]
+            minute_metrics=None,  # type: Optional[Any]
+            cors=None,  # type: Optional[Any]
+            **kwargs  # type: Any
+    ):
+        # type: (...) -> None
+        """Sets properties for an account's Table service endpoint,
+        including properties for Analytics and CORS (Cross-Origin Resource Sharing) rules.
+
+       :param cors: Cross-origin resource sharing rules
+       :type cors: Any
+       :param minute_metrics: Minute level metrics
+       :type minute_metrics: Any
+       :param hour_metrics: Hour level metrics
+       :type hour_metrics: Any
+       :param analytics_logging: Properties for analytics
+       :type analytics_logging: Any
+       :keyword callable cls: A custom type or function that will be passed the direct response
```

Comment:
for the type: Any above, aren't these specific types? e.g. ~azure.table.TableAnalyticsLogging?